buildkite/teams.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2014 Mark Wolfe. All rights reserved.
```

Comment:
This copyright notice doesn't seem accurate.
Perhaps just remove it completely?

Reply:
Yeah I was following what I saw in other files and recent PRs, but agree we should remove it. Thanks!